### DISCUSSION TITLE
Slidy javascript doesn't work on github pages

### DISCUSSION THREAD

**Discussion by lb42** [community/community#21926] [Other Feature Feedback, Questions, & Ideas]
<p>I use the w3c slidy library (<a href="https://www.w3.org/Talks/Tools/Slidy2" rel="nofollow noopener">https://www.w3.org/Talks/Tools/Slidy2</a> ) to create slides for a presentation, and would like to serve the resulting XHTML file from my <a href="http://github.io" rel="nofollow noopener">github.io</a> site</p>
<p>Although the file is correctly rendered for the most part, none of the page handling (display of slides one by one, etc.) works. Is this because the W3C javascript library uses a feature you don’t support, or am I misunderstanding?</p>
<p>Thanks for any advice you can offer.</p>

**Comment by lee-dohm** [comment]
<p>JavaScript is processed in the browser, not on the server. So there isn’t anything that Pages has to support to make JavaScript work (other than transferring the JavaScript file to your browser when requested but that’s all HTTP stuff). Can you link to your Pages site or repo so that we can take a look?</p>

**Comment by lb42** [comment, marked as answer]
<p>Thanks for reply. I found the error : I had a mix of https and http calls in my content . Tsk tsk.</p>
